targets/TARGET_STM/hal_tick_common.c

```diff
@@ -0,0 +1,18 @@
+#include "hal/us_ticker_api.h"
```

Comment:
Can you add a license header on top of this file?

Reply:
Yep. Forgot it...